### PR TITLE
Modulesync 0.12.5 & Release 1.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Change Log
+
+## 2016-09-08 Release [v1.0.0](https://github.com/voxpupuli/puppet-jolokia/tree/v1.0.0)
+
+**Merged pull requests:**
+
+- modulesync 0.11.1 [\#4](https://github.com/voxpupuli/puppet-jolokia/pull/4) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (alexjfisher) [inline CHANGELOG.md]: When automatically generated changelogs attack?

Comment (alexjfisher) [inline CHANGELOG.md]: This is the *first* release right?

**Comment by alexjfisher** [comment]
Perhaps also a README cleanup before 1.0.0 which will be the very first release?

Specifically https://github.com/voxpupuli/puppet-jolokia#development and https://github.com/voxpupuli/puppet-jolokia#release-notescontributorsetc-optional sections

**Comment by alexjfisher** [comment]
In metadata.json, stdlib requirement probably isn't quite accurate ;)
```
"dependencies": [
    {
      "name": "puppetlabs-stdlib",
      "version_requirement": ">= 1.0.0"
    }
  ],
```

I'd also be worth specifying the puppet >4 requirement.

**Comment by alexjfisher** [comment]
https://github.com/voxpupuli/puppet-jolokia/pull/6

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [inline CHANGELOG.md]: yup

File: README.md [hunk 2]
```diff
@@ -102,4 +102,4 @@ Since your module is awesome, other users will want to play with it. Let them kn
 
 ## Release Notes/Contributors/Etc **Optional**
 
-If you aren't using changelog, put your release notes here (though you should consider using changelog). You may also add any additional sections you feel are necessary or important to include here. Please use the `## ` header.
```
Comment (alexjfisher) [inline README.md]: Or delete the whole line/section? ;)

**Pull request merged**